src/data/queries/index.ts

```diff
@@ -13,6 +13,7 @@ import * as AudienceTopics from './audienceTopics'
 import * as Alert from './alert'
```

Comment:
just noting i alphabetized the paragraph queries in addition to adding the two for wysiwyg

Reply:
Thank you!